src/pve_exporter/collector.py

```diff
@@ -244,6 +244,51 @@ def collect(self): # pylint: disable=missing-docstring
 
         return itertools.chain(metrics.values(), info_metrics.values())
 
+class ClusterNodeConfigCollector(object):
+    """
+    Collects Proxmox VE VM information directly from config, i.e. boot, name, onboot, etc.
+    For manual test: "pvesh get /nodes/<node>/<type>/<vmid>/config"
+
+    # HELP pve_vm_config_onboot Proxmox vm config onboot value
+    # TYPE pve_vm_config_onboot gauge
+    pve_vm_config_onboot{id="qemu/113",node="XXXX",type="qemu"} 1.0
+    """
+
+    def __init__(self, pve):
+        self._pve = pve
+
+    def collect(self): # pylint: disable=missing-docstring
+        metrics = {
+            'onboot': GaugeMetricFamily(
+                'pve_vm_config_onboot',
+                'Proxmox vm config onboot value',
+                labels=['id', 'node', 'type']),
```

Comment:
The prometheus metric naming scheme is something like <prefix>_<name>_<unit>(_<aggregation>), e.g. `node_cpu_seconds_total`. So I guess my preferred name would be `pve_onboot_status`. The `vm` prefix is not necessary in my opinion because that one is given by the labels. Looking on other exporters for naming examples I find that node exporter has `node_timex_sync_status`. Ceph has metrics `ceph_health_status`, `ceph_mgr_status` and `ceph_mon_quorum_status`. So it looks like `status` might be a good unit for boolean flags.

Reply:
Didn't know about this and I make the change as you suggest.

Reply:
I only learned about that when they changed a big bunch of metric names in [node exporter](https://github.com/prometheus/node_exporter/releases/tag/v0.16.0). There is also a [post](https://www.robustperception.io/on-the-naming-of-things) by Brian Brazil and some [docs](https://prometheus.io/docs/practices/naming/) on the prometheus site about naming.